src/corner_kick/README.md

```diff
@@ -10,15 +10,16 @@ Corner Kick is a web app that allows for the visualization of UBC Thunderbots' A
 
 **Table of content**
 
--   [How to use](#how-to-use)
--   [Contribute](#contribute)
-    -   [Running the dev server](#running-the-dev-server)
--   [Project Structure](#project-structure)
--   [Design notes](#design-notes)
-    -   [Approach to the application UI layout](#approach-to-the-application-ui-layout)
-        -   [React Portals](#react-portals)
-    -   [Approach to the application state](#approach-to-the-application-state)
-        -   [Reducer functions and Redux saga](#reducer-functions-and-redux-saga)
+- [How to use](#how-to-use)
+- [Contribute](#contribute)
+  - [Running the dev server](#running-the-dev-server)
+- [Project Structure](#project-structure)
+- [Design notes](#design-notes)
+  - [Yarn](#yarn)
+  - [Approach to the application UI layout](#approach-to-the-application-ui-layout)
+  - [Approach to the application state](#approach-to-the-application-state)
+    - [Debugging the Redux store](#debugging-the-redux-store)
+    - [Reducer functions and Redux saga](#reducer-functions-and-redux-saga)
```

Comment:
In general our convention is to use 4 spaces, and people's editors are likely setup to insert that many. Is there a particular reason that we're changing to two spaces here (aside from making the raw markdown easier to read)?